packages/e2e-tests/plugins/interactive-blocks/directive-bind/block.json

```diff
@@ -0,0 +1,14 @@
+{
+	"apiVersion": 2,
+	"name": "test/directive-bind",
+	"title": "E2E Interactivity tests - directive bind",
+	"category": "text",
+	"icon": "heart",
+	"description": "",
+	"supports": {
+		"interactivity": true
+	},
+	"textdomain": "e2e-interactivity",
+	"viewScript": ["directive-bind-view", "wp-interactivity"],
```

Comment:
Why do you need to add `wp-interactivity` here if you are adding it as a dependency when you register the script?

Reply:
IIRC, I had problems with script registering and enqueuing and tested different combinations. 😅 Let me check if that's required or not (it shouldn't, as you mentioned).

Reply:
Correct, it's not needed. I removed `wp-interactivity` from all `viewScript` props.

Reply:
Ok, thank you!